test/account/paymaster/PaymasterCore.test.js

```diff
@@ -68,6 +58,7 @@ for (const [name, opts] of Object.entries({
         );
 
     return {
+      helper,
```

Comment:
Is this and other `helpers` returned from fixtures needed or just kept for convenience?

I'd remove them if they're not used.